src/net.h

```diff
@@ -59,6 +59,8 @@ static const unsigned int MAX_SUBVERSION_LENGTH = 256;
 static const int MAX_OUTBOUND_CONNECTIONS = 8;
 /** Maximum number of addnode outgoing nodes */
 static const int MAX_ADDNODE_CONNECTIONS = 8;
+/** Number of feeler connections */
+static const int NUM_FEELER_CONNECTIONS = 1;
```

Comment:
nit: `MAX_FEELER_CONNECTIONS` for consistency, and coherence with `nMaxFeeler`

Reply:
Right, that makes sense. This essentially becomes 0 when maxconnections < MAX_ADDNODE_CONNECTIONS, which I wasn't sure of initially.